siliconcompiler/schema/schema_cfg.py

```diff
@@ -222,19 +222,7 @@ def schema_fpga(cfg):
     ''' FPGA configuration
     '''
 
```

Comment:
need to update the version in the schema to '0.35.0'

Reply:
Done.